docs/lib-documentation.md

```diff
@@ -250,6 +217,13 @@ Gets the IP address of the Docker container that the service is running inside.
 
 The service's IP address.
 
+### getSharedDirectory() -\> [SharedPath][sharedpath]
+Gets the shared directory's [SharedPath][sharedpath] object who has been dynamically generated during the service bootstrap process.
```

Comment:
Recommend `Get the directory that is mounted on both the current container running this code and the service container, so that files can be passed back and forth. The directory is expressed as a [SharedPath][sharedpath] object, so file inside can be referenced by absolute filepath on either this container or the service contianer.`